### PR TITLE
[manga-cam] Wait for 'loadeddata' event before calculating video size.

### PR DESCRIPTION
**Pull request by lostsource**
Looks like videoWidth / videoHeight were not being properly retrieved.

Resolves GoogleChrome/chrome-app-samples#380

### CODE REVIEW

File: samples/manga-cam/js/index.js [hunk 1]
```diff
@@ -156,10 +153,12 @@ function init(saveImage) {
...
     }
 
-    navigator.webkitGetUserMedia({video: true}, function (stream) {
+    navigator.getUserMedia({video: true}, function (stream) {
       video.src = URL.createObjectURL(stream);
       video.play();
-      setTimeout(function () {
+
+      // wait for enough data to read videoWidth/videoHeight properties
+      video.addEventListener('loadeddata',function(){
```
Comment (beaufortfrancois) [inline samples/manga-cam/js/index.js]: Nit: `function(){` to be replaced with `function() {`

**Comment by beaufortfrancois** [comment]
Thank you for fixing this. A small nit and we're good to go!

**Comment by lostsource** [comment]
Nit corrected and version bumped

**Comment by beaufortfrancois** [comment]
WebStore Item has been updated at https://chrome.google.com/webstore/detail/cnehcenehakkickcnoiaemcpjpaigmbb/
Thank you Joseph!

**Pull request merged**